internal/fuzzbinder/fuzzclaim.go

```diff
@@ -180,3 +181,122 @@ func parseFuzzClaimBytes(statementBytes []byte) (*intoto.Statement, error) {
 
 	return &statement, nil
 }
+
+// generateFuzzClaimSpec generates a fuzzing claim specification using the
+// fuzzing reports of OSS-Fuzz.
+func generateFuzzClaimSpec(date string, revisionDigest intoto.DigestSet, fuzzParameters *FuzzParameters, fuzzTargets []string) (*FuzzClaimSpec, error) {
+	var fuzzClaimSpec FuzzClaimSpec
+	var projectCrashes Crash
+	var projectFuzzEffort FuzzEffort
+	fuzzersCrashes := make(map[string]*Crash)
+	fuzzersFuzzEffort := make(map[string]*FuzzEffort)
+	fuzzersCoverage := make(map[string]*Coverage)
+	hyphenDate := fmt.Sprintf("%s-%s-%s", date[:4], date[4:6], date[6:])
+	//Get fuzzing statistics.
+	projectCoverage, err := GetCoverage(date, fuzzParameters, "perProject", "")
```

Comment:
It is better to refactor the code so that you don't have to pass a magic string like "perProject" to `GetCoverage`.

In particular, you could separate between `GetTotalCoverage` and `GetCoverageForTarget`. Whatever these two functions have in common should be factored into separate function they both call.

Reply:
I added a `// TODO(#171): Split GetCoverage into GetTotalCoverage and GetCoverageForTarget.`

Reply:
I added a related issue to rename functions that take a log of computing: `TODO(#172): Rename functions that take a lot of computation.`